oaknational/Oak-Web-Application
# fix: add webinar image url to seo props

## Description

- get seo image from webinar 

## Issue(s)

Fixes #966 

## How to test

1. Go to {cloud link}
2. Go to webinar page
3. You should see correct og:image head tag
OR
go to vercel link (unprotected)
go to webinar page
paste that link into https://metatags.io/
shoulld see correct description and image


## Checklist
